DEMO_MAINTAINERS.md

```diff
@@ -0,0 +1,33 @@
+# Demo Maintenance
+
+We have several [demo applications](https://github.com/google/generative-ai-docs/tree/main/demos/palm) hosted in this repository that are referenced in the [AI for Developers](https://ai.google.dev/develop/sample-apps) site. We're looking to the community to help maintain them. 
+Thank you in advance for your contributions!
+
+## Responsibilities
+
+While we would love to accept any meaningful contributions to the demos, some tasks we'd particularly like help with include:
+1. Create a process to verify that the app works as desired after any changes are made (preferably automated, but a manual testing process works well to start)
+2. Get dependencies up-to-date
+3. Review and fix any outstanding issues and PRs (you can filter by the `demos:XYZ` [label](https://github.com/google/generative-ai-docs/labels?q=demos%3A))
+4. Migrate from PaLM to Gemini
+
+## Next Steps
+
+If you're interested and commited to maintaining one of the demos, please complete the following:
+- Read through the [Contributing Guide](https://github.com/google/generative-ai-docs/blob/main/CONTRIBUTING.md)
+- Submit a PR to
+  - Add your GitHub handle next to the demo you're interested in maintaining, in the section below
+  - Add your GitHub handle to the [CODEOWNERS](https://github.com/google/generative-ai-docs/blob/main/CODEOWNERS) file if you want to be notified of all new PRs
```

Comment:
IIUC, a code-owner has to be someone with write access to the repository - is that right?

If so, I don't think we want to give write-access to external folks without some proven history first. I don't mind if this means we (I?) have to review changes for a while.

Reply:
That makes sense! I've removed this step.
I left the codeowners file as is (didn't add back previous maintainers).